controllers/binding/snapshotenvironmentbinding_controller.go

```diff
@@ -75,29 +75,26 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 
 	application, err := r.getApplicationFromSnapshotEnvironmentBinding(ctx, snapshotEnvironmentBinding)
 	if err != nil {
-		logger.Error(err, "Failed to get Application for ",
-			"SnapshotEnvironmentBinding.Name ", snapshotEnvironmentBinding.Name, "SnapshotEnvironmentBinding.Namespace ", snapshotEnvironmentBinding.Namespace)
+		logger.Error(err, "Failed to get Application from the napshotEnvironmentBinding")
```

Comment:
typo: `napshotEnvironmentBinding` -> `snapshotEnvironmentBinding`

Reply:
fixed, I like `napshot` though, it sounds like something to put me instantly into a nap